src/main/java/org/neocoder/AuthService.java

```diff
@@ -0,0 +1,14 @@
+package org.neocoder;
+
+import org.mindrot.jbcrypt.BCrypt;
+
+public class AuthService {
```

Comment:
You need a private constructor to avoid instantiating the class incorrectly:

```java
package org.neocoder;

import org.mindrot.jbcrypt.BCrypt;

public class AuthService {

    private AuthService() { }

    public static String hashPassword(String password) {
        return BCrypt.hashpw(password, BCrypt.gensalt());
    }

    public static boolean comparePassword(String password, String hashedpassword) {
        return BCrypt.checkpw(password, hashedpassword);
    }
}
```